operator-common/src/test/java/io/strimzi/operator/common/OperatorMetricsTest.java

```diff
@@ -199,12 +241,18 @@ protected Status createStatus() {
                 .onComplete(context.succeeding(v -> context.verify(() -> {
                     MeterRegistry registry = metrics.meterRegistry();
 
+                    assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "reconciliations").meter().getId().getTags().get(1), is(Tag.of("selector", "")));
                     assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "reconciliations").tag("kind", "TestResource").counter().count(), is(1.0));
+                    assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "reconciliations.successful").meter().getId().getTags().get(1), is(Tag.of("selector", "")));
                     assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "reconciliations.successful").tag("kind", "TestResource").counter().count(), is(1.0));
+                    assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "reconciliations.failed").meter().getId().getTags().get(1), is(Tag.of("selector", "")));
                     assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "reconciliations.failed").tag("kind", "TestResource").counter().count(), is(0.0));
+                    assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "reconciliations.locked").meter().getId().getTags().get(1), is(Tag.of("selector", "")));
                     assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "reconciliations.locked").tag("kind", "TestResource").counter().count(), is(0.0));
+                    assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "resources.paused").meter().getId().getTags().get(1), is(Tag.of("selector", "")));
                     assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "resources.paused").tag("kind", "TestResource").gauge().value(), is(1.0));
 
+                    assertThat(registry.get(AbstractOperator.METRICS_PREFIX + "reconciliations.duration").meter().getId().getTags().get(1), is(Tag.of("selector", "")));
```

Comment:
Same as above ... but also, why do we have the selector always empty here unlike the previous test cases? If you think that we do not need to check for the proper labels / tags in every of these tests, it might be ok. But I wonder if having the empty tag test really makes sense here in that case or should be just removed.

Reply:
In these cases we are not passing any selector label. So the selector label is always going to be empty only. I guess this assert is fine in the cases where we are not passing any Labels and keeping them null

Reply:
I didnt check for empty, null and valued label in all the test, Only the failing reconcile and successful reconcile once. But even if I dont check for labels in the leftiver test it means the labels their are null, so we will definitely get a selector label as value ` selector :  "" `